website/docs/concurrency_control.md

```diff
@@ -71,16 +71,26 @@ hoodie.write.lock.hivemetastore.table
 
 **`Amazon DynamoDB`** based lock provider
 
-Amazon DynamoDB based lock provides a simple way to support multi writing across different clusters
+Amazon DynamoDB based lock provides a simple way to support multi writing across different clusters.  You can refer to the
+[DynamoDB based Locks Configurations](https://hudi.apache.org/docs/configurations#DynamoDB-based-Locks-Configurations)
+section for the details of each related configuration knob.
 
 ```
 hoodie.write.lock.provider=org.apache.hudi.aws.transaction.lock.DynamoDBBasedLockProvider
-hoodie.write.lock.dynamodb.table
-hoodie.write.lock.dynamodb.partition_key
-hoodie.write.lock.dynamodb.region
-hoodie.write.lock.dynamodb.endpoint_url
-hoodie.write.lock.dynamodb.billing_mode
-```
+hoodie.write.lock.dynamodb.table (required)
+hoodie.write.lock.dynamodb.partition_key (optional)
+hoodie.write.lock.dynamodb.region (optional)
+hoodie.write.lock.dynamodb.endpoint_url (optional)
+hoodie.write.lock.dynamodb.billing_mode (optional)
```

Comment:
Why no mark configs versioned_docs with `required/optional` as well?

Reply:
Good catch!  They should be applied to versioned_docs as well.  Fixed.